companion/src/modeledit/expodialog.cpp

```diff
@@ -240,7 +240,7 @@ void ExpoDialog::fmInvertAll()
 {
   lock = true;
   for (int i=0; i<CPN_MAX_FLIGHT_MODES; i++) {
-    cb_fp[i]->setChecked(not cb_fp[i]->checkState());
+    cb_fp[i]->setChecked(!cb_fp[i]->checkState());
```

Comment:
`!cb_fp[i]->isChecked()`

checkState() returns an enum value for tristate checkboxes.  isChecked() returns a Boolean.